### ISSUE TITLE
Could not find nonce in bucket xxx

### ISSUE THREAD

**Issue by liesauer** [handshake-org/hs-airdrop#49]
1. i am sure i have had >=15 followers on GitHub on Feb 4, 2019
2. i am sure i have three SSH public keys associated with my account and never changed.

i tried three of my keys, but still got the same error.

![image](https://user-images.githubusercontent.com/8676741/74890702-cd831400-53bf-11ea-910e-40d0b38d04b4.png)

**Comment by liesauer** [comment]
The final snapshot of the tree was created in Feb. 2019. i have two keys are definitely required.

**Comment by ripzery** [comment]
There's ongoing issue here https://github.com/handshake-org/hs-airdrop/issues/35

**Comment by liesauer** [comment]
> ~There's an ongoing issue here #35~ it was locked, so this will probably be locked too with the same reason 😞

😞so, no reason? just "missing data"? or can i say that is a design fault of HNS?  actually i have claimed lots of airdrops, including keybase XLM. this one is....ummm...

**Comment by pinheadmz** [comment]
#35 was locked because users did not qualify for the airdrop, so it's not an issue, and users were proposing hard forks to add new keys to the airdrop tree, which is off-topic.

This type of case where it seems a user DOES qualify is new, so it will stay open as long as the comments stay on-topic.

I'm not sure the best way to debug, please be patient. 

In the meantime @liesauer please make sure you have the latest version of hs-airdrop, everything installed correctly without errors and that your nodejs version is >=10.0

**Comment by liesauer** [comment]
@pinheadmz yes, i clone the latest code, my ubuntu node is v12 and v10.15 on my imac, and i just upgraded to v12 a few minutes ago, still got the same error.

**Comment by liesauer** [comment]
![image](https://user-images.githubusercontent.com/8676741/74906742-d9d19600-53ec-11ea-8f7b-62ffea92ff4c.png)

**Comment by cuitianze** [comment]
i got the same error

**Comment by devSC** [comment]
me too.

**Comment by fsgmhoward** [comment]
Same error happens for me (and some of my friends who are eligible for this airdrop also), though I was using PGP key.

[My PGP key](https://api.github.com/users/fsgmhoward/gpg_keys) has been added to my Github account on Sep 15, 2017 and has never been changed since then. I have some commits (e.g. [this one](https://github.com/fsgmhoward/shadowsocks-py-mu/commit/6cc8692c8c93c0a091874aa56c80ecd97de6a542)) in 2018 as well as 2019 used this key to sign. I definitely had more than 15 followers on GitHub on Feb 4, 2019.

Yet it shows me that my key cannot be found in bucket 200.

Just a note, I did not have any SSH keys bound to my account since I never use SSH to sync my repositories. However that does not seem to be why my key was not in the tree.

**Comment by boymanjor** [comment]
It is worth noting that only the top 3 most popular [key algorithms](https://github.com/handshake-org/hs-airdrop#accepted-key-algorithms) used on GitHub were accepted. N.b. the RSA modulus and exponent size restrictions. If you have tried all of your acceptable SSH and/or PGP keys, and you still receive the error, you were probably not included in the airdrop tree. I'm not sure why, but there is nothing we can do now that the chain is live.

**Comment by liesauer** [comment]
ummm...okkk...sounds reasonable...

**Comment by boyXiong** [comment]
me too. so sad.

**Comment by whitespur** [comment]
me too.

**Comment by Azard** [comment]
me too.

**Comment by mxito3** [comment]
feel pity.

**Comment by liubo404** [comment]
me too.
1.i am sure i have had >=15 followers on GitHub on Feb 4, 2019
2.i am sure i have three SSH public keys associated with my account and never changed.
 
`Attempting to create proof.
This may take a bit.
Decrypting nonce...
NonceError: Could not find nonce in bucket 230.
    at findNonces (/Users/liubo/tmp/hs-airdrop/bin/hs-airdrop:275:17)
    at async createKeyProofs (/Users/liubo/tmp/hs-airdrop/bin/hs-airdrop:328:17)
    at async main (/Users/liubo/tmp/hs-airdrop/bin/hs-airdrop:746:8)

This means your key was not found in the airdrop tree.
If you registered at handshake.org, use that address.
Usage: $ hs-airdrop [addr]`

**Comment by boymanjor** [comment]
If `hs-airdrop` does not find a nonce, your key is not in the airdrop tree, and you will not receive any coins. Unfortunately, there is nothing we can do about it.

**Comment by boymanjor** [comment]
We already have another one of these threads open here: https://github.com/handshake-org/hs-airdrop/issues/35, so I'm going to close this one.
